source/openapi-data-api-v1.yaml

```diff
@@ -0,0 +1,991 @@
+                    $groupBy: {
+                      "_id": "$status",
+                      "count": { $sum: 1 },
+                      "tasks": { $push: "$text" }
+                    }
+                  },
+                  {
+                    $sort: { "count": 1 }
+                  }
+                ]
+        '400':
+          description: Bad Request
+          $ref: "#/components/responses/BadRequestError"
+        '401':
+          description: Unauthorized
+          $ref: "#/components/responses/UnauthorizedRequestError"
+
+components:
+  securitySchemes:
+    AccessToken:
+      type: apiKey
+      in: header
+      name: Authorization
+      description: |-
+        A [user access
+        token](https://www.mongodb.com/docs/atlas/app-services/users/sessions/)
+        specified as a Bearer token in the Authorization header.
+
+        **Example:** `Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9...`
+    Email:
+      type: apiKey
+      in: header
+      name: email
+      description: |-
+        A registered email/password user's email address. Must be used
+        in conjunction with the ``password`` header.
+
+        **Example:** `email: someone@example.com`
+    Password:
+      type: apiKey
+      in: header
+      name: password
+      description: |-
+        A registered email/password user's password. Must be used in
+        conjunction with the ``email`` header.
+
+        **Example:** `password: Pa55w0rd!`
+    ApiKey:
+      type: apiKey
+      in: header
+      name: apiKey
+      description: |-
+        A server or user API key.
+
+        **Example:** `apiKey: A1g70fN7oKMNCnDRNeyou20Wd24rD6QNmXWYMPwz0SnDRQRQSYwGPdC3y44aJN3H`
+    CustomJwt:
+      type: apiKey
+      in: header
+      name: jwtTokenString
+      description: |-
+        A custom JSON web token string that the App is configured to support.
+
+        **Example:** `jwtTokenString: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJteWFwcC1hYmNkZSIsInN1YiI6IjY0MzZlMDZiZGU1YzQ2ZGQ5MTQwZWFlOSIsIm5hbWUiOiJKb2huIERvZSIsImlhdCI6MTUxNjIzOTAyMn0.Y_6JR4ocrqoCEY3yOZlEqhxtAsSpvlzrXc9uvTSJA0k`
+  responses:
+    BadRequestError:
+      description: The request was malformed or incomplete.
+      content:
+        application/json:
+          schema:
+            oneOf:
+              - $ref: "#/components/schemas/ErrorNoAuthenticationSpecified"
+              - $ref: "#/components/schemas/ErrorMissingAuthenticationParameter"
+
+    UnauthorizedRequestError:
+      description: The authenticated user does not have permission to access this endpoint.
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/ErrorUserNotFound"
+
+  schemas:
+
+    # Query Operations
+    Namespace:
+      type: object
+      required: ["dataSource", "database", "collection"]
+      properties:
+        dataSource:
+          type: string
+          description: The name of a linked MongoDB Atlas data source.
```

Comment:
is this always "mongodb-atlas"? even if it's possible to config it to smthn else, might be worth noting in the description that it's `"mongodb-atlas"` by default.